pkg/cloudprovider/provider/kubevirt/provider.go

```diff
@@ -0,0 +1,329 @@
+package kubevirt
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strconv"
+
+	"github.com/kubermatic/machine-controller/pkg/cloudprovider/cloud"
+	cloudprovidererrors "github.com/kubermatic/machine-controller/pkg/cloudprovider/errors"
+	"github.com/kubermatic/machine-controller/pkg/cloudprovider/instance"
+	"github.com/kubermatic/machine-controller/pkg/providerconfig"
+
+	corev1 "k8s.io/api/core/v1"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"github.com/Azure/go-autorest/autorest/to"
```

Comment:
Can you create a refactoring issue to create a dedicated package for such pointer helper functions? More a "make me feel less dirty" refactoring

Reply:
I removed the dependency on that package from the kubevirt provider instead